pod.go

```diff
@@ -144,4 +158,16 @@ func (pc *podCollector) collectPod(ch chan<- prometheus.Metric, p v1.Pod) {
 		addGauge(descPodContainerStatusReady, boolFloat64(cs.Ready), cs.Name)
 		addCounter(descPodContainerStatusRestarts, float64(cs.RestartCount), cs.Name)
 	}
+
+	nodeName := p.Spec.NodeName
+	for _, c := range p.Spec.Containers {
+		req := c.Resources.Requests
+		cpu := req[v1.ResourceCPU]
+		mem := req[v1.ResourceMemory]
+
+		addGauge(descPodContainerRequestedCpuMilliCores, float64(cpu.MilliValue()),
+			c.Name, nodeName)
```

Comment:
Above you declared the labels
```
"namespace", "pod", "container", "node"
```
but you are setting `namespace` to `c.Name` and `pod` to `nodeName`. Same below.